app/src/main/cpp/skyline/services/btm/IBtmUserCore.cpp

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2022 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#include "IBtmUserCore.h"
+
+namespace skyline::service::btm {
+    IBtmUserCore::IBtmUserCore(const DeviceState &state, ServiceManager &manager) : BaseService(state, manager) {}
+
+    Result IBtmUserCore::AcquireBleScanEvent(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        response.Push(std::make_shared<type::KEvent>(session.state, false));
```

Comment:
This isn't how events are done, see other examples, also add comment on bool to clarify meaning

Reply:
(to clarify you need to keep event as member var and return a handle rather than pointer)

Reply:
Mind giving a look over the changes?